<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotPropsWidget</class>
 <widget class="QWidget" name="RobotPropsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Robot position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="horizontalSliderAngle">
        <property name="maximum">
         <number>359</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPosX"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPosY"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditAngle"/>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
